src/tools/pub/installing.md

```diff
@@ -1,49 +0,0 @@
----
```

Comment:
Why are we getting rid of this page?

Reply:
Because we don't _install_ pub. One installs the Dart SDK; the main pub page opens with:

> You can use the `pub` tool to manage Dart packages. Pub is one of the tools that you get when you [install](/tools/sdk#install) the Dart SDK.

In think that that is all we need.

The rest of the pub "installing" page was actually only about environment variable configuration. Which is what I renamed to page to.